src/seaport.ts

```diff
@@ -130,6 +131,185 @@ export class OpenSeaPort {
     this._emitter.removeAllListeners(event)
   }
 
+  /**
+   * Wraps an arbirary group of NFTs into their corresponding WrappedNFT ERC20 tokens.
+   * Emits the `WrapAssets` event when the transaction is prompted.
+   * @param param0 __namedParameters Object
+   * @param tokenIds Token IDs of the assets.
+   * @param tokenAddresses Addresses of the tokens' contracts. Must be the same length as `tokenIds`. Each address corresponds with its respective token ID in the `tokenIds` array.
+   * @param accountAddress Address of the user's wallet
+   */
+  public async wrapAssets(
+      { tokenIds, tokenAddresses, accountAddress }:
+      { tokenIds: string[];
```

Comment:
Apologies for the inconsistency that you might have noticed, but we're changing this syntax to `assets: Asset[];` instead of two separate arguments for `tokenIds` and `tokenAddresses`. See `createBundleSellOrder`